packages/backend/src/SynthqlError.ts

```diff
@@ -135,6 +135,32 @@ export class SynthqlError extends Error {
 
         return new SynthqlError(new Error(), type, lines.join('\n'), 404);
     }
+
+    static createPermissionsError({
+        node,
+        missingPermissions,
+        contextPermissions,
+    }: {
+        node: AnyQuery;
+        missingPermissions: string[];
+        contextPermissions: string[];
+    }) {
+        const type = 'PermissionsError';
+
+        const lines = [
+            `The query '${node?.name}' with a permissions list (ACL) included:`,
```

Comment:
simplify this even more:

`The query ${query.name} is missing the following permissions: ${missingPermissions}`

The first line should have most of the useful information, use the remaining lines for additional context